api/get_user_visit_count.go

```diff
@@ -28,6 +28,6 @@ func GetUserVisitCount(remoteAddr string, params interface{}, c *gin.Context) (r
 	// get openbbsmiddleware user count
 	currentUserVisitCount := schema.GetUserVisitCount()
 	// total user
-	result = GetUserVisitCountResult{int64(result_b.Total) + currentUserVisitCount}
+	result = GetUserVisitCountResult{Total: int64(result_b.Total) + currentUserVisitCount}
```

Comment:
hi @tingyuchang ～


Instead of GetUserVisitCountResult, I prefer to have

`result = &GetUserVisitCountResult{}`

It is because that it is considered as non-primitive types and it's better to use pointer for non-primitive types.

Reply:
ok, I misunderstand previous request change, you mean `&` NOT `{Total: xx}`, sometimes IDE will notify me should use explicit naming, so I thought you said the same one.
I know using pointer is better for performance and avoid pass by value to copy data.

Reply:
hi @tingyuchang ～

Sorry about the previous confusion.
I mean

`&GetUserVisitCountResult{Total: ...}`

Which is both `&` and `{Total: ...}`

Reply:
it's ok, thanks for suggestion~